packages/stokhos/test/UnitTest/Stokhos_UnitTestHelpers.hpp

```diff
@@ -142,10 +148,10 @@ namespace Stokhos {
 
     // Compare elements
     for( int i = 0; i < n; ++i ) {
-      ValueType err = std::abs(a1.coeff(i) - a2.coeff(i));
+      ValueType err = KAT::abs(a1.coeff(i) - a2.coeff(i));
```

Comment:
Why did you change?  `Kokkos::ArithTraits` is being replaced by Kokkos math functions and numeric traits.
This function is not annotated with `KOKKOS_FUNCTION` so `std::abs` is just fine.  If there was an ambiguity, then cast the arguments as necessary.